package.json

```
@@ -110,7 +110,9 @@
       "eslint:recommended",
```

The order of the plugins goes from the most generic (eslint) to the most specific (typescript ) for rules to be override
```suggestion
 "extends": [
      "eslint:recommended",
      "plugin:vue/essential",
      "@vue/typescript"
    ],
    ```
  
https://eslint.vuejs.org/user-guide/#configuration